lib/core_extensions.rb

```diff
@@ -115,7 +115,7 @@ def self.unconfigured?
   def self.per_page
     # Foreman.in_rake? prevents the failure of db:migrate for postgresql
     # don't query settings table if in rake
-    return 20 if Foreman.in_rake?
+    return 20 unless Foreman.settings.ready?
     Setting[:entries_per_page] rescue 20
   end
```

Comment:
I believe we can actually get rid of this monkeypatch and just set
```
WillPaginate.per_page = Setting[:entries_per_page]
```
in the will_paginate initializer that runs after the settings have already been loaded.

Reply:
That would prevent us from changing this value without restart of foreman though.

Reply:
There would have to be an after_safe hook, that would set the value `WillPaginate.per_page` after the setting page and that is out of scope of this PR. I believe this is the same what we had before, we don't need pagination from setting in initialization process, so the result will be fine IMHO.

Reply:
This should probably move to `ApplicationRecord` which would fix the issue but doesn't have to be in the same pr.